lti_consumer/templates/html/student.html

```diff
@@ -22,28 +32,41 @@ <h2 class="problem-header">
 
 % if launch_url and not hide_launch:
     % if launch_target in ['modal', 'new_window']:
-        <section class="wrapper-lti-link">
-            % if description:
-                <div class="lti-description">${description}</div>
-            % endif
-            <p class="lti-link external">
-                % if launch_target == 'modal':
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-modal"
-                        data-target="#${element_id + '-lti-modal'}"
-                    >
-                        ${button_text or 'View resource in a modal window'} <i class="icon fa fa-external-link"></i>
-                    </button>
-                % else:
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-new-window"
-                        data-target="${form_url}"
-                    >
-                        ${button_text or 'View resource in a new window'} <i class="icon fa fa-external-link"></i>
-                    </button>
+        <section
+            class="wrapper-lti-link"
+            aria-label="${_('This section of the courseware is delivered by a 3rd party. As such, we cannot make any guarantees that this external resource supports the same level of accessibility.')}"
+        >
+            % if launch_pane_html:
+                ## it is intended that authors can use HTML in the launch_pane_html parameter
+                ## so we allow this to go through unescaped
+                ${launch_pane_html | n, decode.utf8}
+            % else:
+                % if description:
+                    ## I believe historically, we have some courses which pipe through some
+                    ## HTML in the description field, so let's preserve existing behavior
+                    <div class="lti-description">${description | n, decode.utf8}</div>
+                % endif
+                <p class="lti-link external">
```

Comment:
What do the classes `lti-link` and `external` do?  Do you need them with the custom html?

The reason I was looking at this is because my manual testing found the following defect:
The custom pane HTML with new window (versus modal) seems to be broken.

Also, what will happen if the class on the button doesn't properly match the "LTI Launch Target" setting?  Do we need a textual reminder to have them match?  It would be better if the author didn't have to be concerned with all this, and if something could be done (maybe in JavaScript) to simply add the appropriate class based on the LTI Launch Target.